### ISSUE TITLE
Feature request or doc to save others some time - easy infinite scroll instead of pager

### ISSUE THREAD

**Issue by andrewblake1** [kartik-v/yii2-grid#379]
Not sure if you are aware how easy infinite scroll can replace paging in your grid view widget:
```
        'pager' => [
            'class' => \kop\y2sp\ScrollPager::className(),
            'container' => '.grid-view',
            'item' => '.kv-grid-table tbody tr',
            'triggerOffset' => 1000,
        ],
```
works easy to add infinite scroll. After investigating various extensions and plugins, tried this one and worked first go (so far anyway - with only one minor display anomaly on spinner after filtering).

Potentially could add option to gridview perhaps - or just a note or demo in docs might help others.

**Comment by kartik-v** [comment]
Thanks for the suggestion. Will look at it when I get time.

**Comment by andrewblake1** [comment]
Don’t rush, there are a couple of minor issues when used with pjax that I will look into in next few days - i think there are events in the plugin that will fix these but I will found out and put explanation and solution back in this issue.

**Comment by kartik-v** [comment]
yes as mentioned will need to evaluate once I get time for this... its currently not on my priority list

**Comment by andrewblake1** [comment]
Understood, and thanks for so many great extensions. They are a massive asset to the framework.

**Comment by andrewblake1** [comment]
FYI when you get time:

My final personal configuration:
```
        'pager' => [
            'class' => ScrollPager::className(),
            'container' => '.grid-view',
            'item' => '.kv-grid-table tbody tr',
            'enabledExtensions' => [
                ScrollPager::EXTENSION_SPINNER,
            ],
        ],
```
There also appears to be a minor easily by-passable bug when used with pjax at this time though could be fixed by the time you get this - my for now solution and suggestion for fix is at https://github.com/webcreate/infinite-ajax-scroll/issues/219

**Comment by strtob** [comment]
Hi,

Unfortunately, I got this error in terms of bs4:

`Setting unknown property: kop\y2sp\ScrollPager::linkContainerOptions`

Is it possible to make it work ?

Thanks a lot!

**Comment by maxxer** [comment]
For the record, I used different selectors (still on bs3)

```php
    'pager' => [
        'class' => ScrollPager::class,
        'container' => '.kv-grid-table tbody',
        'item' => 'tr',
        'enabledExtensions' => [
            ScrollPager::EXTENSION_SPINNER,
        ],
    ],
